src/effects/VST3/VST3Effect.cpp

```diff
@@ -1233,3 +1177,54 @@ void VST3Effect::ReloadUserOptions()
 
    SetBlockSize(mUserBlockSize);
 }
+
+
+bool VST3Effect::TransferDataToWindow(const EffectSettings& settings)
+{
+   const auto vst3Settings = GetSettings(settings);
+
+   const bool scanOK = ForEachParameter(
```

Comment:
I think I must rewrite this method, because it is not doing what is supposed to, i.e. take the input settings and move sliders and knobs on the plugin GUI to reflect their values. 

What it is doing instead, is only passing on the settings to the audio processor (through the ComponentHandler).